src/shogun-core-main/src/main/java/de/terrestris/shoguncore/util/interceptor/secure/WfsRequestInterceptor.java

```diff
@@ -131,6 +134,20 @@ public MutableHttpServletRequest interceptTransaction(MutableHttpServletRequest
     private boolean isAllowed(
         MutableHttpServletRequest request, String paramName, String method) {
         String typeNameParam = request.getParameterIgnoreCase(paramName);
+        if (typeNameParam == null) {
+            // try to get typename from POST body, if any
+            try {
+                String body = IOUtils.toString(request.getInputStream(), StandardCharsets.UTF_8.name());
+                if (!StringUtils.isEmpty(body) && StringUtils.containsIgnoreCase(body, "wfs:transaction")) {
```

Comment:
The `wfs` xml namespace can be changed to nearly any string AFAICT. This should probably be enhanced, at least a TODO or follow up issue should be doable, correct?

Reply:
Issue created: https://github.com/terrestris/shogun-core/issues/400